File: Python/Aksesoris.py
from Petshop import Petshop

class Aksesoris(Petshop):
    def __init__(self, id, nama_produk, kategori_produk, harga, jenis, bahan, warna):
        super().__init__(id, nama_produk, kategori_produk, harga)
        self.Jenis = jenis
        self.Bahan = bahan
        self.Warna = warna

    # Setter
    def setJenis(self, jenis):
        self.Jenis = jenis
    def setBahan(self, bahan):
        self.Bahan = bahan
    def setWarna(self, warna):
        self.Warna = warna

    # Getter
    def getJenis(self):
        return self.Jenis
    def getBahan(self):
        return self.Bahan
    def getWarna(self):
        return self.Warna

File: Java/Aksesoris.java
class Aksesoris extends Petshop{
    private String Jenis;
    private String Bahan;
    private String Warna;

    public Aksesoris(int id, String nama_produk, String kategori_produk, int harga, String jenis, String bahan, String warna) {
        super(id, nama_produk, kategori_produk, harga);
        this.Jenis = jenis;
        this.Bahan = bahan;
        this.Warna = warna;
    }

	// Setter
    void setJenis(String jenis){this.Jenis = jenis;}
	void setBahan(String bahan){this.Bahan = bahan;}
	void setWarna(String warna){this.Warna = warna;}

	// Getter
    String getJenis(){return Jenis;}
    String getBahan(){return Bahan;}
    String getWarna(){return Warna;}
}